src/plugins/query_enhancements/server/connections/managers/prometheus_manager.ts

```diff
@@ -94,11 +115,8 @@ class PrometheusManager extends BaseConnectionManager<OpenSearchClient> {
     query: PrometheusResourceQuery
   ): Promise<GetResourcesResponse<R>> {
     return this.getClient(context, request).getResources<R>({
-      query,
       path: `${URI.DIRECT_QUERY.RESOURCES}/${query.dataSourceName}/${this.getResourceURI(query)}`,
-      params: {
-        querystring: new URLSearchParams(query.query).toString(),
-      },
+      querystring: new URLSearchParams(query.query).toString(),
```

Comment:
Can you verify this doesn't break the existing usage in Discover (for fetching labels and metrics)?

Reply:
tested with /resources/metric_metadata and /resources/labels that all worked